File: problems/b-tax/tax.cpp
#include <iostream>

using namespace std;

int main () {
	int t;
	cin >> t;
	while (t--) {
		int x;
		cin >> x;
		int withheld = x * 0.5;
		int finale = 0;
		if (x > 10000) finale += (x - 10000) * (0.1 - 0.0);
		if (x > 20000) finale += (x - 20000) * (0.2 - 0.1);
		if (x > 40000) finale += (x - 40000) * (0.4 - 0.2);
		if (x > 80000) finale += (x - 80000) * (0.8 - 0.4);
		int balance = finale - withheld;
		if (balance < 0) cout << "Refund " << -balance << endl;
		else if (balance > 0) cout << "Owe " << balance << endl;
		else cout << "Clear" << endl;
	}
	return 0;
}

File: problems/b-tax/tax.py
n = int(input())

for i in range(n):
    total = int(input())
    held = total // 2
    owed = 0
    if total > 80000:
        owed += (total - 80000) * 4 // 5
        total = 80000
    if total > 40000:
        owed += (total - 40000) * 2 // 5
        total = 40000
    if total > 20000:
        owed += (total - 20000) // 5
        total = 20000
    if total > 10000:
        owed += (total - 10000) // 10
        total = 10000
    if owed == held:
        print('Clear')
    elif owed > held:
        print('Owe %d' % (owed - held))
    else:
        print('Refund %d' % (held - owed))
